test/resources.e2e-spec.ts

```diff
@@ -0,0 +1,424 @@
+import { Test, TestingModule } from "@nestjs/testing";
+import { INestApplication, ValidationPipe } from "@nestjs/common";
+import { seed } from "../prisma/seed/seed";
+import * as request from "supertest";
+import { AppModule } from "../src/app.module";
+import { PrismaService } from "../src/prisma/prisma.service";
+import { CreateResourceDto } from "src/resources/dto/create-resource.dto";
+import { UpdateResourceDto } from "src/resources/dto/update-resource.dto";
+import { extractResCookieValueByKey } from "./utils";
+
+const loginUser = async (
+    email: string,
+    password: string,
+    app: INestApplication,
+) => {
+    const res = await request(app.getHttpServer())
+        .post("/auth/login")
+        .send({
+            email,
+            password,
+        })
+        .expect(200);
+
+    return extractResCookieValueByKey(
+        res.headers["set-cookie"],
+        "access_token",
+    );
+};
+
+const findVoyageTeamId = async (email: string, prisma: PrismaService) => {
+    return await prisma.voyageTeamMember.findFirst({
```

Comment:
await is redudant here, and line 45 (sorry my IDE is giving me a warning and I don't like seeing it 😂)